src/core_write.cpp

```diff
@@ -166,6 +168,16 @@ void TxToUniv(const CTransaction& tx, const uint256& hashBlock, UniValue& entry)
             o.pushKV("asm", ScriptToAsmStr(txin.scriptSig, true));
             o.pushKV("hex", HexStr(txin.scriptSig.begin(), txin.scriptSig.end()));
             in.pushKV("scriptSig", o);
+            if (!tx.wit.IsNull()) {
+                if (!tx.wit.vtxinwit[i].IsNull()) {
+                    UniValue txinwitness(UniValue::VARR);
+                    for (unsigned int j = 0; j < tx.wit.vtxinwit[i].scriptWitness.stack.size(); j++) {
```

Comment:
We're trying to move away from BOOST in favour of c++11. See https://github.com/bitcoin/bitcoin/projects/3